utils/zsh/patches/010-use-libpcre2.patch

```diff
@@ -0,0 +1,638 @@
+--- a/Src/Makefile.in
```

Comment:
Each patch, which is getting added, usually in the ideal world should contain **Git header**.
Git header includes important details like: Commit author, Commit description, Commit subject, etc.

This helps to know who authored such changes and where it comes from. Is that backport, which will be removed with upcoming release.

Reply:
So it should list myself as author, and description that says that Src/Makefile.in is patched to support openwrt which lacks AC_PROG_LN and just has AC_PROG_LN_S? I'd rather patch this to macros of auto-tools, so it would include AC_PROG_LN as well, but that would be one more PR; Even though bible says, that man should fill the country with his offsprings; it shouldn't be task for just one man.. If someone adds that to auto-tools defs, I'll gladly remove this change.

Reply:
Hmm, yes, could be. Sounds good